spec/amp-html-layout.md

```diff
@@ -120,13 +120,17 @@ Supported values for the `layout` attribute:
       <td><code>flex-item</code></td>
       <td>The element and other elements in its parent with layout type <code>flex-item</code> take the parent container's remaining space when the parent is a flexible container (i.e., <code>display: flex</code>). The <code>width</code> and <code>height</code> attributes are not required.</td>
     </tr>
+    <tr>
+      <td><code>intrinsic</code></td>
+      <td>The element takes the space available to it and resizes its height automatically to the aspect ratio given by the <code>width</code> and <code>height</code> attributes. This layout works very well for most AMP elements, including <code>amp-img</code>, <code>amp-video</code>, etc.  The available space depends on the parent element and can also be customized using <code>max-width</code> CSS. The <code>width</code> and <code>height</code> attributes must be present.  This layout differs from <code>responsive</code> by having an intrinsic height and width. This is most apparent inside a floated element where a <code>responsive</code> layout will render 0x0 and <code>intrinsic</code> layout will inflate to the smaller of it's natural size or any CSS constraint. </td>
```

Comment:
The fragment `and can also be customized using <code>max-width</code> CSS.` does that pertain to the parent element or to the element that has the `intrinsic` layout? Please clarify wording.